### PR TITLE
UI

### PR DESCRIPTION
**Pull request by BunnyPrince**
# General info

**Issue number**: 89

**Task description**: 

- move the notification box into the main dashboard box
- center the whole content
- add link to the logo
- add footer
- change background color
- stack header element in small screen size

 
 
Updated
image
image
image
image

# Testing
no test for the ui

### CODE REVIEW

**Comment by mth-1012** [comment]
Finally it looks fine on 1440p screen. Some stuffs I noticed (visually):

- Can you change this...

image

...to match the other headers (like Today's News or Symptoms)

- Should add more footer space and reduce footer font size (takes a bit too much space with 1080p screen)

- Might be a nitpick but I think texts on red background should be white instead of black, it's a bit difficult to look at (at least for me, I'm not colorblind I believe)

image

**Comment by BunnyPrince** [comment]
image
image
image


for the empty notification I see that there is already a code written for it but I don't know why is not showing up (the file NotificationBox.js)
image

File: frontend/src/components/NotificationBox/NotificationBox.js [hunk 1]
```diff
@@ -68,9 +68,28 @@ function NotificationBox() {
...
+
+            {/*<div className={styles["notification-container"]}>*/}
+            {/*    {emptyMessage ? <EmptyMessage/> : null}*/}
+            {/*    {notifications.map((notification) => (*/}
+            {/*        <div className={styles["notification-box"]} key={notification._id}>*/}
+            {/*            <Notification*/}
+            {/*                notificationId={notification._id}*/}
+            {/*                timeStamp={notification.timeStamp}*/}
+            {/*                alertType={notification.type}*/}
+            {/*                alertHeading={notification.heading}*/}
```
Comment (AC159) [inline frontend/src/components/NotificationBox/NotificationBox.js:83]: Commented code should be removed

File: frontend/src/components/NotificationBox/NotificationBox.module.css [hunk 2]
```diff
@@ -5,21 +5,11 @@
 }
 
 .notification-container {
```
Comment (AC159) [inline frontend/src/components/NotificationBox/NotificationBox.module.css:7]: Remove empty css classes

**Review by AC159 — APPROVED** [review]
Looks much better!

**Review by DrakeHunterK7 — APPROVED** [review]
The UI overhaul has come together beautifully. Great work, @BunnyPrince! 

This PR can be merged now.

**Pull request merged**